neutone_sdk/audio.py

```diff
@@ -46,24 +104,13 @@ def to_metadata_format(self):
         }
 
 
-def audio_sample_to_mp3_bytes(sample: AudioSample) -> bytes:
-    buff = io.BytesIO()
-    with tempfile.NamedTemporaryFile(suffix=".mp3") as temp:
-        torchaudio.save(temp.name, sample.audio, sample.sr)
-        with open(temp.name, "rb") as f:
-            buff.write(f.read())
-    buff.seek(0)
-    return buff.read()
-
-
 def audio_sample_to_mp3_b64(sample: AudioSample) -> str:
-    mp3_bytes = audio_sample_to_mp3_bytes(sample)
+    mp3_bytes = sample.to_mp3_bytes()
     return base64.b64encode(mp3_bytes).decode()
 
 
 def mp3_b64_to_audio_sample(b64_sample: str) -> AudioSample:
```

Comment:
Same here but `from_mp3_b64_str()` or similar

Reply:
I left this as from_b64 as it could take in wav as well